frontend/mr-admin-flate/src/components/modal/SletteModal.tsx

```diff
@@ -33,22 +40,18 @@ const SletteNotatModal = ({
     onClose();
   };
 
-  const handleDelete = () => {
-    mutation.mutate(notatIdForSletting, { onSuccess: onClose });
-  };
-
   function headerInnhold() {
     return (
       <div className={styles.heading}>
         {mutation.isError ? (
           <>
             <ExclamationmarkTriangleFillIcon className={styles.erroricon} />
-            <span>Kan ikke slette notatet.</span>
+            <span>{headerTextError}</span>
           </>
         ) : (
           <>
             <XMarkOctagonFillIcon className={styles.warningicon} />
-            <span>Ønsker du å slette notatet?</span>
+            <span>{headerText}</span>
```

Comment:
Denne kan nok være en form for heading og ikke span. For mer korrekt semantisk html i modalen